### DISCUSSION TITLE
Should nitro preset from nuxt.config be used in nuxi dev? or only in nuxi build?

### DISCUSSION THREAD

**Discussion by Lundi** [nuxt/nuxt#17713] [Questions] [answered]
I've set a nitro preset in my `nuxt.config.ts` to be used when building my project for deployment, like so:
```
  nitro: {
    preset: 'aws-lambda',
    serveStatic: true,
  }
  ```

Now with this configuration when I run `nuxi dev` I run into `[worker reload] [worker] exited` errors as seemingly Nitro is building for aws-lambda and not a node dev server. Removing this setting allows `nuxi dev` to run normally again.

Is this expected behaviour? Or should `nuxi dev` always use a `node-server` NITRO_PRESET regardless of whether it's set in `nuxt.config.ts` or not?

**Comment by pi0** [comment, marked as answer]
Hi. Issue will be solved by https://github.com/unjs/nitro/commit/94fc53118e411bc538d61dacac70d1d1183b5004 in next nitro release shortly.
